apollo-execution-processor/src/main/kotlin/com/apollographql/execution/processor/definitions.kt

```diff
@@ -434,8 +429,12 @@ private class TypeDefinitionContext(
       }
 
       ClassKind.INTERFACE -> {
-        if (!modifiers.contains(Modifier.SEALED)) {
-          logger.error("Interfaces and unions must be sealed interfaces", this)
+        val isIgnored = hasIgnoreQuery
```

Comment:
This was not quite working, maybe not a good approach either 😅 Is there a better way? Maybe some way to get annotations when doing the walk function?